data_pipeline/data_processing/aggregatedcsv2redis.py

```diff
@@ -42,8 +44,10 @@ def __init__(self, filename: str):
             logger.info('{} only have {} row.'.format(
                 filename, self.data.shape[0]))
         # Add candidate ID column. candidate ID is <Ballot Item>;<CandidateControlledName>;<Election Date>
-        self.data['ID'] = self.data['Ballot Item'].map(
-            str) + ';' + self.data['CandidateControlledName'].str.replace(' ', '-') + ';' + self.data['Election Date'].map(str)
+        self.data['Ballot Item'] = self.data['Ballot Item'].str.replace('-', ' ')
+        self.data['ID'] = self.data['Ballot Item'].str.replace(
+            ' ', '_') + ';' + self.data['CandidateControlledName'].str.replace(' ', '_') + ';' + self.data[
+                              'Election Date'].map(str)
```

Comment:
Is it ok to keep the '/' in the election date? I don't remember if we still need to replace them w/ something else i.e. 11/3/2020 -> 11-3-2020

Reply:
Removing forward slash in the candidate ID will allow us to query a single candidate by /candidates/:ID. However, we need to sync up with the front-end team on what format would be good, ex. 11-3-2020 or 20201103. We can address it in a separate commit.

Reply:
ok, i'll just approve and merge for now